tests/core/metering.py

```
@@ -101,17 +104,13 @@ def get_all_events(self, event_type: str, resource_id: str) -> list[dict[str, An
     def _fetch_events(self, event_type: str, resource_id: str) -> list[dict[str, Any]]:
-        params = urlencode({
-            "type": event_type,
-            "resource_id": resource_id,
-            "since": self._start_time,
-        })
+        params = urlencode({"type": event_type, "resource_id": resource_id, "since": self._start_time})
         url = f"{self._base_url}/events?{params}"
         last_exc: OSError | None = None
         for attempt in range(3):
             try:
                 req = Request(url)
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)